server_test.go

```diff
@@ -0,0 +1,60 @@
+package goodman
+
+import (
+	"bufio"
+	"net"
+	"testing"
+)
+
+func TestSendingServerMessages(t *testing.T) {
+	runner := DummyRunner{}
+	server := NewServer([]Runner{&runner})
+
+	go func() {
+		err := server.Run()
+		if err != nil {
+			t.Fatalf("Dredd hooks server failed to start with error %s", err.Error())
+		}
+	}()
+
+	messages := []struct {
+		Payload []byte
+	}{
+		// TODO: Figure why I could not use `` quoted string
```

Comment:
`` should have worked fine here. Let me know if you want a hand to figure out what went wrong.